src/cljam/io/vcf/reader.clj

```diff
@@ -64,10 +64,26 @@
         => {:id \"NS\" :number \"1\" :type \"Integer\"
             :description \"Number of Samples With Data\"}"
   [s]
-  (->> (re-seq #"([\w:/\.\?\-]*)=(?:(?:\"([^\"]*?)\")|([\w:/\.\?\-]*))" s)
-       (map (fn [[_ k v1 v2]]
-              {(->kebab-case-keyword k) (dot->nil (or v2 v1))}))
-       (apply merge)))
+  (letfn [(parse-string-field [s]
```

Comment:
Great implementation but it looks like a bit too complex to me 🤔 

Regexp like following will do the job except throwing the exceptions, right? or is it not enough?
```clojure
(->> (re-seq #"([\w:/\.\?\-]*)=(?:(?:\"((?:[^\\\"]|\\\"|\\\\)*)\")|([\w:/\.\?\-]*))\s*(?=(?:,|$))" s)
     (into {} (map (fn [[_ k v1 v2]]
                     [(->kebab-case-keyword k) (cstr/replace (dot->nil (or v2 v1)) #"\\(?=[\\\"])" "")]))))
```

Reply:
> it looks like a bit too complex

Definitely. I think so, too 😂

As far as I'm aware, the only difference between your implementation and mine is just error checking, as you told. But, I'm suspecting the regex approach would scale enough to parse `##META` line in the future. I don't want to see that monolithic regex more complex :sob: 
I think the ideal implementation may be in between the two.

Reply:
> But, I'm suspecting the regex approach would scale enough to parse ##META line in the future

OK, let's leave it as it is until the spec of `##META` line gets clarified.
Thanks!